src/main/markdown/doc/latest/tutorial/JSON.md

```diff
@@ -272,18 +269,18 @@ The new StockData class will be an overlay type which overlays the existing Java
 ```
 package com.google.gwt.sample.stockwatcher.client;
 
-    import com.google.gwt.core.client.JavaScriptObject;
+import com.google.gwt.core.client.JavaScriptObject;
 
-    class StockData extends JavaScriptObject {                              // <span style="color:black;">**(1)**</span>
+class StockData extends JavaScriptObject {                              // <span style="color:black;">**(1)**</span>
```

Comment:
Would you mind changing `// <span style="color:black;">**(1)**</span>` to `// (1)` and making sure these comments start on the same column (there are a few more below)?

Currently it renders bad as well: http://www.gwtproject.org/doc/latest/tutorial/JSON.html

Reply:
I didn't change them because I didn't know which formatting to choose. You proposal is a good idea. Just fixed it up.